opentelemetry/src/propagation/mod.rs

```diff
@@ -1,170 +1,30 @@
 //! # OpenTelemetry Propagator interface
+//! Cross-cutting concerns send their state to the next process using Propagators, which are defined
+//! as objects used to read and write context data to and from messages exchanged by the applications.
 //!
-//! Propagators API consists of two main formats:
+//! `Propagator`s leverage the [`Context`] to inject and extract data for each cross-cutting concern,
+//! such as `TraceContext` and [`Baggage`].
 //!
-//! - `BinaryFormat` is used to serialize and deserialize a value
-//! into a binary representation.
-//! - `TextMapFormat` is used to inject and extract a value as
-//! text into injectors and extractors that travel in-band across process boundaries.
+//! The Propagators API is expected to be leveraged by users writing instrumentation libraries.
 //!
-//! Deserializing must set `is_remote` to true on the returned
-//! `SpanContext`.
+//! Currently, the following `Propagator` types are supported:
+//! -  [`TextMapPropagator`], inject values into and extracts values from carriers as string key/value pairs
 //!
-//! ## Binary Format
+//! A binary Propagator type will be added in
+//! the future, See [tracking issues](https://github.com/open-telemetry/opentelemetry-specification/issues/437)).
 //!
-//! `BinaryFormat` is a formatter to serialize and deserialize a value
```

Comment:
Do we want to remove all this documentation?

Reply:
I think it mostly came from old specification which can be misleading. We can add stuff like `BinaryPropagator` back once we implement it